wordle-web/tests/game.test.ts

```diff
@@ -0,0 +1,28 @@
+// @vitest-environment nuxt
+import { expect, test } from "vitest";
+import { Game } from "~/scripts/game";
+import { LetterState } from "~/scripts/letter";
+
```

Comment:
Even though not said in the requirements, I'd recommend making another test just to get used to vitest syntax.  Creating new tests helped me get a better understanding of the framework.

Reply:
Yes, good idea. Meant to do that, but didn't get around to it before the submission.